odxtools/basicstructure.py

```diff
@@ -411,6 +422,17 @@ def param_sort_key(param: Parameter) -> Tuple[int, int]:
             formatted_lines.append(next_line)
             next_line = ""
 
+            if 0 < param_idx < len(params) - 1 and hasattr(
```

Comment:
I prefer range checks to be oldschool:
```suggestion
            if 0 < param_idx and param_idx < len(params) - 1 and hasattr(
```

(while `a < x < b`  works with python it feels too magical to me because `(a < x) < b` will *not* work in all cases)...

Reply:
fixed